### PR TITLE
v1.0.0 deployed addresses

### PR DESCRIPTION
**Pull request by fleupold**
🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,15 +220,16 @@ tenderly actions deploy
...
-| `TradeAboveThreshold`          | TBD                                                                                                                   | TBD                                                                                                                  | TBD                                                                                                                    |
-| `StopLoss`          | TBD                                                                                                                   | TBD                                                                                                                  | TBD                                                                                                                    |
+| Contact Name                   | Ethereum Mainnet                                                                                                      | Goerli                                                                                                                       | Gnosis Chain                                                                                                           |
```
Comment (mfw78) [inline README.md:223]:
```suggestion
| Contract Name                  | Ethereum Mainnet                                                                                                      | Goerli                                                                                                                       | Gnosis Chain                                                                                                           |
```

**Review by mfw78 — APPROVED** [review]
Looks great, only a typo nit! 🛳️🛳️

File: README.md (hunk 1 above)
Comment (mfw78) [reply README.md]: Fixed.

**Review by mfw78 — APPROVED** [review]

**Comment by anxolin** [comment]
NVM! the issue was in the Web. It was using the new ComposableCow with the old TWAP contract 🙈
emarasing enough, i checked the PR and i thought all addresses were good, so then when i saw the code one matched the one seen by WatchTower, i though, OK twap is the right one but points to the old ComposableCoW!

It was the old TWAP old along.

Thx @mfw78 and @fleupold for the help

**Pull request merged**